lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -88,7 +88,10 @@ class TraceOfTab extends ComputedArtifact {
       firstMeaningfulPaint = lastCandidate;
     }
 
-    const onLoad = keyEvents.find(e => e.name === 'MarkLoad' && e.ts > navigationStart.ts);
+    const onLoad = frameEvents.find(e => e.name === 'loadEventStart' && e.ts > navigationStart.ts);
```

Comment:
markload should actually line up with loadEventEnd. if its not, then it must be selecting the wrong frame.

loadEventEnd is the dfn we want to use, so let's adjust to that instead.